packages/app/src/containers/LandingPage/index.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import {
+    ImageBackground,
+    Image,
+    Text,
+    View,
+    StyleSheet
+} from 'react-native';
+import { Button } from "react-native-elements";
+
+interface LandingPageState {
+}
+
+interface OwnProps {
+    navigation: any
+}
+
+type Props = OwnProps;
+
+class LandingPage extends React.Component<Props, LandingPageState> {
+    constructor(props: Props) {
+        super(props);
+
+        this.state = {
+        };
+    }
+
+
+    public componentDidMount() {
+    }
+
+    public render() {
+
+        return (
+            <React.Fragment>
+                <ImageBackground source={require("../../../assets/images/bg.jpg")} resizeMode="cover" style={styles.container}>
```

Comment:
this path doesnt exist in the repo, image is not found

Reply:
@petermsouzajr  Images folder did not push to the repo because of gitignore. Now I have renamed the folder and it is fixed now